templates/agenda/index.html.twig

```diff
@@ -101,14 +105,15 @@
         </div>
     </main>
 
-
-    <footer class="col-12 mt-3">
-        <div class="row">
-            <div class="col-auto ml-auto">
-                <a name="" id="" class="btn btn-success" href="create" role="button">{% trans %}Create agenda{% endtrans %}</a>
+    {% if not is_granted('ROLE_BOARD_MEMBER') %}
```

Comment:
Use `{% if is_granted('ROLE_CASEWORKER') %}`

Reply:
Fixed in 4bc1a7723936a2511ae7d457d4dfec56d37c7bf1.